src/apurvaBabel/Assignment20.java

```diff
@@ -0,0 +1,124 @@
+package apurvaBabel;
+
+public class Assignment20 {
+
+	// Method to return sum of all non prime numbers from given array
+	int getSumOFNonPrimeNumbers(int[] num) {
+		int sum = 0;
+		for (int index = 0; index < num.length; index++) {
+			if (num[index] > 0) {
+				for (int i = 2; i < num[index] / 2; i++) {
```

Comment:
here in condition, we need to put <= condition, rest thing looks quite good

Reply:
Rectified in PR 828
![image](https://user-images.githubusercontent.com/112226757/190082416-e998d06f-b315-4c32-bd4f-42a185a2f96b.png)